### PR TITLE
Switch forecast.io to darksky.net

### PR DESCRIPTION
**Pull request by joshgachnang**
Forecast.io is going away, and they request a "powered by" thing.

### CODE REVIEW

File: forecastio/forecastio.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 angular.module('mirror')
     .factory('Weather', function ($q, $resource, $http, FORECASTIO_KEY) {
-        var url = 'https://api.forecast.io/forecast/' + FORECASTIO_KEY + '/';
+        var url = 'https://api.darksky.net/forecast/' + FORECASTIO_KEY + '/';
```
Comment (Kurt-von-Laven) [inline forecastio/forecastio.js]: Seems like `FORECASTIO_KEY` should be renamed.

**Review by Kurt-von-Laven — CHANGES_REQUESTED** [review]
Not sure if there are other things that should be renamed.

**Comment by joshgachnang** [comment]
Whoops, missed pushing this last commit, which is why the forecast is broken. PTAL.

**Review by Kurt-von-Laven — APPROVED** [review]
Ah hah! **LGTM**

**Pull request merged**